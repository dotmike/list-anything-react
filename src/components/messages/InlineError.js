import React from "react";
import PropTypes from "prop-types";

const style = {
  color: "#ae5856",
  textAlign: "left",
  minWidth: "100%"
};

const InlineError = ({ text }) => <div style={style}>{text}</div>;

InlineError.propTypes = {
  // ownProps
  text: PropTypes.string.isRequired
};

export default InlineError;
