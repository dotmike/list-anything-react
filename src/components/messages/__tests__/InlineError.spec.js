import React from "react";
import { shallow } from "enzyme";

import InlineError from "../InlineError";

describe("InlineError", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<InlineError text="some text" />);
    expect(wrapper).toMatchSnapshot();
  });
});
