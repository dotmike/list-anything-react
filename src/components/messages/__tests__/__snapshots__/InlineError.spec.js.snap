// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineError renders correctly 1`] = `
<span
  style={
    Object {
      "color": "#ae5856",
    }
  }
>
  some text
</span>
`;
